rectools/visuals/visual_app.py

```diff
@@ -470,7 +724,7 @@ class ItemToItemVisualApp(VisualAppBase):
 
     def __init__(
         self,
-        recos: TablesDict,
+        recos: tp.Union[pd.DataFrame, tp.Dict[tp.Hashable, pd.DataFrame]],
```

Comment:
```suggestion
        recos: tp.Union[pd.DataFrame, TablesDict],
```

Reply:
Left on purpose to clear things up for end users. I can change it to TabledDict if you think it's more clear. Like we are using ExternalIds for example.

Reply:
I think it's more clear, but if you feel it's not, you can keep it as is

Reply:
Decided to keep TablesDict everywhere